hack/olm.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# olm.sh - run/cleanup the operator with OLM
+#
+# USAGE
+#    olm.sh -a run/cleanup -c container_url:tag
+# OPTIONS
+#    -a=     Action                   run/cleanup the operator installation
+#    -i      Ignore image cache       builds the operator image without using local image build cache
+#    -c=     Container:Tag            container url and tag for the operator image
+
+
+# container tool to use with operator-sdk
+CONTAINER_TOOL=podman
+
+function error-exit() {
+    echo "Error: $*" >&2
+    exit 1
+}
+
+# Options
+while getopts ":ia:c:" opt; do
+    case "$opt" in
+	a) ACTION="$OPTARG" ;;
+	i) noCache="--image-build-args=\"--no-cache\"";;
+	c) OPERATOR_IMAGE="$OPTARG";;
+	?) error-exit "Unknown option"
+    esac
+done
+
+if [[ ! "$ACTION" =~ ^run|cleanup$ ]]; then
+    error-exit "-a Action must be \"run\" or \"cleanup\""
+fi
+
+if [ -z "$AWS_SHARED_CREDENTIALS_FILE" ]; then
+    error-exit "env AWS_SHARED_CREDENTIALS_FILE not found"
+fi
+
+if [ -z "$KUBE_SSH_KEY_PATH" ]; then
+    error-exit "env KUBE_SSH_KEY_PATH not found"
+fi
+
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+source $WMCO_ROOT/hack/common.sh
+
+cd $WMCO_ROOT
+OSDK=$(get_operator_sdk)
+
+# Builds the container image and pushes it to repository
+# Uses this built image to run the operator on the cluster
+# containers are tagged by branch name and it is user's responsibility to clean old/unused containers in
+# container repository as well as local system.
```

Comment:
Please be consistent with your formatting. Why is one line longer than the rest? Stick to 120 column formatting.